File: app/src/main/java/com/arturogutierrez/openticator/domain/account/list/AccountListView.java
package com.arturogutierrez.openticator.domain.account.list;

import com.arturogutierrez.openticator.domain.account.model.AccountPasscode;
import java.util.List;

public interface AccountListView {

  void viewNoItems();

  void renderAccounts(List<AccountPasscode> accounts);

  void showEditActionButtons();
}

File: app/src/main/kotlin/com/arturogutierrez/openticator/domain/account/list/AccountListView.kt
package com.arturogutierrez.openticator.domain.account.list

import com.arturogutierrez.openticator.domain.account.model.AccountPasscode
import com.arturogutierrez.openticator.view.presenter.Presenter

interface AccountListView : Presenter.View {

  fun noItems()

  fun renderAccounts(accounts: List<AccountPasscode>)

  fun startEditMode()
}
